apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -254,5 +280,8 @@ class ProgressBubble extends React.Component {
     return bubble;
   }
 }
+connect(state => ({
+  userId: state.currentUser.userId
+}))(ProgressBubble);
```

Comment:
Will this cause issues loading large scripts?

Reply:
@davidsbailey any thoughts here? Doesn't seem to slow down script loads from testing.